### PR TITLE
Fix EltwiseReduceMod

### PR DESCRIPTION
**Pull request by fboemer**
Fixes https://github.com/intel/hexl/issues/86

### CODE REVIEW

File: test/test-eltwise-reduce-mod.cpp [hunk 1]
```diff
@@ -79,5 +79,20 @@ TEST(EltwiseReduceMod, 4_2) {
   CheckEqual(result, exp_out);
 }
 
+// https://github.com/intel/hexl/issues/86
+TEST(EltwiseReduceMod, LargeModError) {
```
Comment (joserochh) [inline test/test-eltwise-reduce-mod.cpp:83]: Maybe it can be called LargeModTest or LargeMod? unless you are expecting an error

Comment (fboemer) [reply test/test-eltwise-reduce-mod.cpp]: I removed it; the (EltwiseReduceModTest, Random) test was sufficient to reproduce the error.

**Comment by fboemer** [comment]
@bryan113, thanks for reporting this. Are you able to check if this PR fixes the SEAL issue https://github.com/microsoft/SEAL/issues/411?

**Review by joserochh — APPROVED** [review]
LG

**Review by jlhcrawford — APPROVED** [review]
Was there a particular reason you didn't choose to parameterise the existing tests?

Also I don't think you needed to change the test fixture name to have the "Test" suffix if you wanted to keep it consistent with the non-parameterised tests.

Otherwise, this looks good to me.

**Comment by bryan113** [comment]
I tested SEAL with HEXL commit: `42f057d` and it appears to be working now. Thank you!

**Comment by WeiDaiWD** [comment]
@fboemer Any plan to release another version of HEXL?

**Comment by fboemer** [comment]
> @fboemer Any plan to release another version of HEXL?

@WeiDaiWD, yes we'll plan to release v1.2.3 with this fix in a day or two.

**Pull request merged**